.github/workflows/ci.yaml

```diff
@@ -63,6 +63,61 @@ jobs:
           bash ./ci/run_tests.sh
           bash ./ci/ensure_copyright.sh
 
+  ubuntu-clang14-address-undefined-leak:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          ssh-key: ${{ secrets.SSH_KEY }}
+
+      - name: Install clang-14
+        run: |
+          sudo apt-get update && \
+          sudo apt-get install build-essential software-properties-common -y && \
```

Comment:
nit. In github actions you don't have to add a `&& \` for each line when using the default shell. It will evaluate the entire run step as a shared shell.

Reply:
Excellent; thanks.